system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_granule_update_schema_test.clj

```diff
@@ -0,0 +1,74 @@
+(ns cmr.system-int-test.ingest.bulk-update.bulk-granule-update-schema-test
+  (:require
+   [cheshire.core :as json]
+   [clojure.string :as string]
+   [clojure.test :refer :all]
+   [cmr.common.util :as util :refer [are3]]
+   [cmr.message-queue.test.queue-broker-side-api :as qb-side-api]
+   [cmr.mock-echo.client.echo-util :as echo-util]
+   [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+   [cmr.system-int-test.system :as sys]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest]))
+
+(use-fixtures :each (ingest/reset-fixture {"provguid1" "PROV1"}))
+
+(def base-request {:name "sample bulk update"
+                   :operation "UPDATE_FIELD"
+                   :update-field "OPeNDAPLink"
+                   :updates
+                   [["ur_1" "s3://aws.foo.com"]
+                    ["ur_2" "s3://aws.bar.com"]]})
+
+(deftest bulk-granule-schema-validation-test
+  (are3
+   [update-fn err-msg]
+   (let [bulk-update-options {:token (echo-util/login (sys/context) "user1")
+                              :accept-format :json
+                              :raw? true}
+         request (update-fn base-request)
+         {:keys [body status]} (ingest/bulk-update-granules "PROV1"
+                                                            request
+                                                            bulk-update-options)
+         response (json/parse-string body true)]
+
+     (is (= 400 status))
+     (is (seq (filter #(= err-msg %) (:errors response)))
+         (format "Error message containing [%s] was not found in [%s]"
+                 err-msg
+                 (pr-str response))))
+
+   "insufficient bulk operation targets"
+   (fn [m] (assoc m :updates []))
```

Comment:
I'd like a comment block explaining the what/why with the `fn` being passed, or for the tests to be made simpler (even if it makes them verbose)